back-end/src/main/java/ro/utcn/sd/cata/stackoverflow/controller/QuestionController.java

```diff
@@ -0,0 +1,43 @@
+package ro.utcn.sd.cata.stackoverflow.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ro.utcn.sd.cata.stackoverflow.command.AddQuestionCommand;
+import ro.utcn.sd.cata.stackoverflow.command.Command;
+import ro.utcn.sd.cata.stackoverflow.dto.CreateQuestionDTO;
+import ro.utcn.sd.cata.stackoverflow.dto.QuestionDTO;
+import ro.utcn.sd.cata.stackoverflow.entity.Question;
+import ro.utcn.sd.cata.stackoverflow.service.QuestionService;
+
+import java.util.List;
+
+@RestController
+public class QuestionController {
+    private final QuestionService questionService;
+
+    @Autowired
+    public QuestionController(QuestionService questionService) {
+        this.questionService = questionService;
+    }
+
+    @GetMapping("/view-all-questions")
```

Comment:
The URLs don't respect at all the REST design principles... https://slides.com/spet/utcn-sd-assignment-3#/3/2